sretoolbox/container/image.py

```diff
@@ -390,7 +396,13 @@ def manifest(self):
         """
         if self._cache_manifest is None:
             manifest = self._get_manifest()
-            self._cache_manifest = manifest.json()
+            try:
+                self._cache_manifest = manifest.json()
+            except json.decoder.JSONDecodeError as exc:
+                raise ImageInvalidManifestError(
```

Comment:
Do we know why is that happening? I see we have the response, but only the JSON conversion is erroing... Is it like incomplete/broken?

Reply:
just curious :)

Reply:
got it, thanks!